src/transform-sdk/tests/test_util.go

```diff
@@ -61,6 +61,7 @@ func startRedpanda(ctx context.Context) (*redpanda.Container, context.CancelFunc
 			req.LogConsumerCfg.Consumers = append(req.LogConsumerCfg.Consumers, &stdoutLogConsumer{})
 			return nil
 		}),
+		testcontainers.WithEnv(map[string]string{"WASMTIME_BACKTRACE_DETAILS": "1"}),
```

Comment:
so this means we get sane backtraces in redpanda logs?

Reply:
No I need to enable more build settings in Wasmtime for sane backtraces I think